### ISSUE TITLE
Would you consider release the pseudo labels predicted by your model for DSEC_Night training split?

### ISSUE THREAD

**Issue by Candy-Crusher** [XiaRho/CMDA#5]
As titled. Since "All experiments are conducted on a Tesla A100 GPU", and due to the limitation of the GPU I can get accessed to, it's hard to reproduce this work.

**Comment by XiaRho** [comment]
Hi, @Candy-Crusher. We understand the constraints you're facing due to GPU limitations. To generate the pseudo labels, you can download the trained model [CMDA for Cityscapes→DSEC](https://drive.google.com/file/d/1-UGTyTN0gfahSFhUKNzXQIwunbxo0Nda/view?usp=sharing).

Once you have the model, you can follow the inference process similar to that of Cityscapes to DarkZurich `python my_run_experiments.py --root_path /path_to_CMDA/CMDA/ --base_config configs/fusion/cs2dz_image+raw-isr_b5.py --name cmda_cs2dz`. Modify a little code and apply this model to the DSEC-Night training set to generate pseudo labels. Rest assured, this process should not require an excessive amount of GPU memory.

**Comment by Candy-Crusher** [comment]
Hi! I was using python my_test.py --work_dir ./CMDA/pretrained/CMDA_DSEC_test_60.05/230301_1700_cs2dsec_image+events_together_A01B0005C1_b5_e978d but got all key in source state_dict unexpected. Could you plz give me some hints?

![image](https://github.com/user-attachments/assets/401ce5bd-a562-47cd-ae35-5fbcb1454d4a)

**Comment by Candy-Crusher** [comment]
My reproduce result is like this:
![image](https://github.com/user-attachments/assets/96631986-2c74-426a-bd2d-a65a27b4bc5b)

Which seems to be align with CMDA(I) but the final miou is different?

<img width="1011" alt="1726044751607" src="https://github.com/user-attachments/assets/2b7359e1-dd86-441b-b17e-2845eae9afe3">

**Comment by XiaRho** [comment]
As we said in the README, the logs provide the mIoU for 19 classes. For Cityscapes→DSEC, it is necessary to convert the mIoU to the 18 valid classes (without the truck class), i.e., the final mIoU 54.72 should be converted to 54.72*19/18=57.76.

**Comment by Candy-Crusher** [comment]
I see, thx for the quick respond! So the key in source state_dict unexpected can be safely ignored right?

**Comment by XiaRho** [comment]
Yes, there is no need for EMA parameter when testing.

**Comment by Candy-Crusher** [comment]
Got it. The code is clear and well-organized, thx for the great work and quick respond!

**Comment by Candy-Crusher** [comment]
![image](https://github.com/user-attachments/assets/99adddf3-69c0-4be9-888d-97297b9b36aa)
May I ask how did you choose the labels_range and why you substract 13 from each of them?

**Comment by XiaRho** [comment]
We took the first part of each sequence as labels in a certain proportion. "substract 13" seems to be related to the selection interval of the labels but is not important.
